go/samples/telemetry-test/kitchen_sink/kitchen_sink.go

```diff
@@ -0,0 +1,165 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/firebase/genkit/go/ai"
+	"github.com/firebase/genkit/go/genkit"
+	"github.com/firebase/genkit/go/plugins/firebase"
+	"github.com/firebase/genkit/go/plugins/googlegenai"
+	"github.com/firebase/genkit/go/plugins/server"
+	"google.golang.org/genai"
+)
+
+type SearchInput struct {
+	Query string `json:"query" description:"The search query or topic to search for"`
+}
+
+type SearchResult struct {
+	Title   string `json:"title"`
+	Content string `json:"content"`
+}
+
+func main() {
+	ctx := context.Background()
+
+	// Initialize Firebase telemetry
+	firebase.EnableFirebaseTelemetry(&firebase.FirebaseTelemetryOptions{
+		ForceDevExport: true, // Force telemetry export in development
+	})
+
+	// Initialize Genkit with plugins
+	g, err := genkit.Init(ctx, genkit.WithPlugins(
```

Comment:
Latest Genkit doesn't return error from here anymore, go.sum is probably outdated?